cinema/views.py

```diff
@@ -34,7 +47,37 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
-    def get_serializer_class(self):
+    @staticmethod
+    def params_to_int(query_string: str) -> List[int]:
+        return [int(param) for param in query_string.split(",")]
+
+    @staticmethod
+    def params_to_str(query_string: str) -> List[str]:
+        return [param for param in query_string.split(",")]
+
+    def get_queryset(self) -> QuerySet[Movie]:
+        queryset = self.queryset
+
+        actors = self.request.query_params.get("actors", None)
+        genres = self.request.query_params.get("genres", None)
+        title = self.request.query_params.get("title", None)
+
+        if actors:
+            actors = self.params_to_int(actors)
+            queryset = queryset.filter(actors__id__in=actors)
+        if genres:
+            genres = self.params_to_int(genres)
+            queryset = queryset.filter(genres__id__in=genres)
```

Comment:
for many to many no need to use __id, just actors__in=actors_ids should be enough